### PR TITLE
This makes the util constants file more legible when it comes to dates

### PR DESCRIPTION
**Pull request by mischat**
... and times ... No semantic change, just makes the code easier to read.

### CODE REVIEW

File: test/util/constants.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // Some periods in milliseconds
```
Comment (cliffhall) [inline test/util/constants.js:1]: Should say seconds instead of milliseconds

**Review by cliffhall — CHANGES_REQUESTED** [review]
Comment needs updating

File: test/util/constants.js (hunk 1 above)
Comment (mischat) [reply test/util/constants.js:1]: yes, pushing now

**Review by hswopeams — APPROVED** [review]

**Review by cliffhall — APPROVED** [review]
LGTM! 👍

**Pull request merged**